llvm/lib/Target/X86/Disassembler/X86Disassembler.cpp

```diff
@@ -1134,6 +1134,25 @@ static int getInstructionIDWithAttrMask(uint16_t *instructionID,
   return 0;
 }
 
+static bool isNF(InternalInstruction *insn) {
+  // NF bit is the MSB of aaa.
+  if (nfFromEVEX4of4(insn->vectorExtensionPrefix[3]) &&
+      insn->opcodeType == MAP4)
+    return true;
+  // Promoted BMI instrs below has nf version but not in map4.
```

Comment:
We'd better distinguish promoted instructions from NF instructions in words. 
"Promoted" means "NF=0, ND =0".
I suggest change the comment to
"// Below NF instructions are not in map4."